docs/src/pages/components/avatars/GroupAvatars.tsx

```diff
@@ -1,17 +1,13 @@
 import React from 'react';
 import Avatar from '@material-ui/core/Avatar';
 import AvatarGroup from '@material-ui/lab/AvatarGroup';
-import Tooltip from '@material-ui/core/Tooltip';
 
 export default function GroupAvatars() {
   return (
     <AvatarGroup>
       <Avatar alt="Remy Sharp" src="/static/images/avatar/1.jpg" />
       <Avatar alt="Travis Howard" src="/static/images/avatar/2.jpg" />
       <Avatar alt="Cindy Baker" src="/static/images/avatar/3.jpg" />
-      <Tooltip title="Foo • Bar • Baz">
```

Comment:
Not in its current form – it duplicates the default functionality this PR introduces.

Reply:
The problem the tooltip is trying to solve is expose the list of all the members in the group. What would be a great answer to it?

Reply:
I'm not sure, but since it doesn't work with the max feature, it's a bit pointless here. Also, while the demo tooltip works with "Foo", "Bar", and "Baz" as names, with 3 (or many) real names, the tooltip would be unwieldy.

Reply:
Yes, the idea would be to disable the max feature. Regarding the size, we have a max-width of 300. It was a cheap alternative to https://atlaskit.atlassian.com/packages/core/avatar-group

<img width="215" alt="Capture d’écran 2020-02-27 à 00 16 38" src="https://user-images.githubusercontent.com/3165635/75397152-7ec10700-58f6-11ea-86ed-c0202fea565f.png">

Reply:
> Yes, the idea would be to disable the max feature.

So, just to be clear, you want to: 
- Introduce a new feature
- Not add a demo for it
- Keep the existing demo showing equivalent functionality
- Disable the new feature in that demo
- Demonstrate a tooltip that isn't practical for the real world use?

Did I miss anything? 😉

Reply:
(Although I have to agree – the Atlaskit example is 🤮)

Reply:
Ok, so we remove it?

Reply:
Remove it, or change it to a demo of the new feature...

Reply:
Ok

Reply:
So the final thought is to keep demo changes I made here where the example is removed?

Reply:
I have no strong preference

Reply:
I'll change to include the `max` prop for functionality and showing only 3/6 avatars